MapboxCoreNavigation/MGLMapboxEvents.swift

```diff
@@ -0,0 +1,30 @@
+import Mapbox
+import Polyline
+
+extension MGLMapboxEvents {
+    public class func addDefaultEvents(routeProgress: RouteProgress, sessionIdentifier: UUID, sessionNumberOfReroutes: Int = 0) -> [String: Any] {
```

Comment:
Is there any way we can share this function with MapboxNavigation without making it public? We need it for `navigation.cancel`